addons/website_sale/models/product.py

```diff
@@ -339,6 +342,21 @@ def _get_current_company_fallback(self, **kwargs):
         website = self.website_id or kwargs.get('website')
         return website and website.company_id or res
 
+    def _init_column(self, column_name):
+        # to avoid generating a single default website_sequence when installing the module,
+        # we need to set the default row by row for this column
+        if column_name == "website_sequence":
+            _logger.debug("Table '%s': setting default value of new column %s to unique values for each row", self._table, column_name)
```

Comment:
Was the logger intended to be deployed like this? If it's the case it should probably just use `info` and not `debug`?

Reply:
tips: you can use `%r` instead of `'%s'`